cfe_internal/enterprise/CFE_knowledge.cf

```diff
@@ -56,6 +56,16 @@ bundle agent cfe_internal_setup_knowledge
       handle => "cfe_internal_setup_knowledge_dir_doc_root",
       perms => mog("0550", "root", $(def.cf_apache_group) );
 
+      "$(sys.workdir)/httpd/php" -> { "ENT-9703" }
+        comment => "php directory needs to be accessible (execute) to cfapache",
```

Comment:
Especially in relation to processing alerts (notifications and custom actions).

Reply:
I think it would be good to add the run alerts promise handle to the promisees.

#bringbacktheknowlegemap